packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -209,6 +209,28 @@ describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDo
         cy.get('[data-cy="confirm"]').click()
       })
     })
+
+    it('window:before:load event', () => {
+      cy.visit('/fixtures/multi-domain.html')
```

Comment:
The `beforeEach` navigates from primary->secondary and then this is navigating back. Is the first part of this test really needed? I wonder if it would make sense to pull these window tests into their own file.

Reply:
You're right that this is navigating back and forth, I do like that this is clearly asserting that the before and after events are being added to their correct windows. I can break the 'events' out into their own test file.